atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -202,6 +209,9 @@ def get_cluster_info(self, cluster, platform):
         conf = Configuration(**kwargs)
         osbs = OSBS(conf, conf)
         current_builds = self.get_current_builds(osbs)
+        if current_builds == self.UNREACHABLE_CLUSTER_LOAD:
```

Comment:
...and maybe catch `OsbsException` here? i.e. wrap `self.get_current_builds(osbs)` in a `try:` block.